### PR TITLE
get-started: add alias for kubectl in minikube

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added, or deleted? (Required)

Add alias for `kubectl` via `minikube`

The `--` is needed to pass arguments to `kubectl` instead of `minikube`

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: en/get-started.md [hunk 1]
```diff
@@ -196,6 +196,14 @@ minikube start --docker-env https_proxy=http://127.0.0.1:1086 \
 
 See [minikube setup](https://kubernetes.io/docs/setup/minikube/) for more options to configure your virtual machine and Kubernetes cluster.
 
+To interact with the cluster `kubectl` is used. This is included as subcommand in `minikube`. The alias below can be used to make this available directly as the rest of the documentation expects. You will have to put this in your shell profile or execute this after opening a new shell.
```
Comment (yikeke) [inline en/get-started.md:199]:
```suggestion
To interact with the cluster, you can use `kubectl`. This is included as a sub-command in `minikube`. The alias below can be used to make this available directly as the rest of the documentation expects. You will have to put this in your shell profile or execute this after opening a new shell.
```

**Review by yikeke — APPROVED** [review]

File: en/get-started.md [hunk 2]
```diff
@@ -196,6 +196,14 @@ minikube start --docker-env https_proxy=http://127.0.0.1:1086 \
 
 See [minikube setup](https://kubernetes.io/docs/setup/minikube/) for more options to configure your virtual machine and Kubernetes cluster.
 
+To interact with the cluster, you can use `kubectl`. This is included as a sub-command in `minikube`. The alias below can be used to make this available directly as the rest of the documentation expects. You will have to put this in your shell profile or execute this after opening a new shell.
```
Comment (qiancai) [inline en/get-started.md:199]: There are four "This" used in the paragraph. To make the content easier to understand, could we rephrase as below? Thanks!

```suggestion
To interact with the cluster, you can use `kubectl`, which is included as a sub-command in `minikube`. To make `kubectl` take effect as the rest of the documentation expects, you can either add the following alias definition command to your shell profile or execute the following alias definition command after opening a new shell.
```

**Pull request merged**